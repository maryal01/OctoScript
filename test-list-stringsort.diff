diff: tests/test-list-stringsort.out: No such file or directory
