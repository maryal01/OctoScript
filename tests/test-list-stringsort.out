adebayor
beckham
hari
krishna
ramayan
rooney
tevez
